askar-crypto/Cargo.toml

```diff
@@ -101,7 +101,7 @@ p384 = { version = "0.13", default-features = false, features = [
     "ecdh",
 ], optional = true }
 rand = { version = "0.8", default-features = false }
-secure-env = { package = "animo-secure-env", version = "0.4", optional = true }
+secure-env = { package = "animo-secure-env", version = "0.5", optional = true }
```

Comment:
Just as a matter of interest — what is the `animo-secure-env`?

Reply:
It allows hardware backed key generation and signing on iOS / android. We should probably have chosen a more descriptive name :)

See https://github.com/animo/secure-env